.github/workflows/spark-rapids-jni.yaml

```diff
@@ -7,7 +7,7 @@ jobs:
   spark-rapids-jni-build:
     runs-on: linux-amd64-cpu8
     container:
-      image: rapidsai/ci-spark-rapids-jni:rockylinux8-cuda12.8.0
+      image: rapidsai/ci-spark-rapids-jni:rockylinux8-cuda12.9.1
```

Comment:
Depends on PR: https://github.com/NVIDIA/spark-rapids-jni/pull/3262

Reply:
filed new https://github.com/NVIDIA/spark-rapids-jni/pull/3484, 

and we are building new images for rapids .github/workflows/spark-rapids-jni.yaml